CONTRIBUTING.md

```diff
@@ -50,3 +50,11 @@ Build: Update Travis to only test Node 0.10 (refs #734)
 Fix: Semi rule incorrectly flagging extra semicolon (fixes #840)
 Upgrade: Esprima to 1.2, switch to using comment attachment (fixes #730)
 ```
+
+## Branch naming convention
+
+The branch should have a name that reflects it's purpose.
+
+The convention is to prefix the branch name with feature-. All the words must be separated by a -.
+
+Branch name example : feature-user-authentication
```

Comment:
Pour les noms de branch je proposerais plus un "/" après le type de branch (feature, fix etc.) 
A voir l'avis général

Ex: feature/user-authentication

Reply:
Same for me i prefer the naming convention with "/"

Reply:
Moi ça me va si tout le monde est d'accord :)

Reply:
Et pour la séparation des mots - ou _ ?

Reply:
"-"